source/partials/drupal-8-convert-to-composer.md

```diff
@@ -0,0 +1,255 @@
+## Apply All Available Upstream Updates
+
+[Update the site](/core-updates) to the latest [Pantheon Drops 8](https://github.com/pantheon-systems/drops-8) Upstream and apply all available updates.
+
+1. Use Terminus to list all available updates:
+
+  ```bash{outputLines:2}
+  terminus upstream:updates:list $SITE
+  [warning] There are no available updates for this site.
+  ```
+
+1. If any updates are available, apply them using the command line or via the [Pantheon Dashboard](/core-updates#apply-upstream-updates-via-the-site-dashboard):
+
+  ```bash{promptUser: user}
+  terminus upstream:updates:apply $SITE.dev --updatedb
+  ```
+
+## Add the Pantheon Integrated Composer Upstream in a New Local Branch
+
+This process involves significant changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back changes can be complicated:
+
+1. In your local terminal, change directories to the site project. For example, if you keep your projects in a folder called `projects` in the home directory:
+
+  ```bash{promptUser:user}
+  cd ~/projects/$SITE/
+  ```
+
+1. Add the Pantheon Drupal Upstream as a new remote called `ic`, fetch the `ic` branch, and checkout to a new local branch based on it called `composerify`:
+
+  ```bash{promptUser:user}
+  git remote add ic git@github.com:pantheon-upstreams/drupal-project.git && git fetch ic && git checkout -b composerify ic/master
```

Comment:
This command causes the `composerify` branch to track `master`:
```
❯ git checkout -b composerify ic/master
Branch 'composerify' set up to track remote branch 'master' from 'ic'.
Switched to a new branch 'composerify'
```

Reply:
You probably want the `--no-track` flag:
```
❯ git checkout --no-track -b newbranch ic/master
Switched to a new branch 'newbranch'
```

Reply:
```
❯ git branch -vv
  composerify e51c478 [ic/master] Pantheon Drupal 9 update
* newbranch   e51c478 Pantheon Drupal 9 update
```

Reply:
added --no-track